vignettes/nestcolor.Rmd

```diff
@@ -0,0 +1,128 @@
+---
+title: "Getting Started with nestcolor"
+date: "2022-07-20"
+output:
+  rmarkdown::html_document:
+    theme: "spacelab"
+    highlight: "kate"
+    toc: true
+    toc_float: true
+vignette: >
+  %\VignetteIndexEntry{Getting Started}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+### Introduction
+
+`nestcolor` is a package containing standardized color palettes used in various NEST package plots and graphical `teal` modules to create a more continuous and cohesive experience when viewing NEST visualizations.
+
+The following NEST packages are optimized to implement `nestcolor`:
+
+- `tern`
+- `chevron`
+- `goshawk`
+- `osprey`
+- `teal.modules.clinical`
+- `teal.modules.general`
+- `teal.goshawk`
+- `teal.osprey`
```

Comment:
Please do not couple hard-coded text with the implementation somewhere else. This section is likely to be outdated in the future. What if we would implement nestcolor in let's say teal.modules.hermes? Do we need to update this vignette then? 

I propose to skip it or make this statement more general. Would be safer and easier to maintain.

Reply:
I have removed this section.